src/main/scala/com/yotpo/metorikku/output/writers/file/FileOutputWriter.scala

```diff
@@ -84,7 +84,17 @@ class FileOutputWriter(props: Map[String, Object], outputFile: Option[File]) ext
             }
             catalog.refreshTable(tableName)
           }
-          case false => writer.saveAsTable(tableName)
+          case false => {
+            writer.save()
+            catalog.createTable(tableName, filePath)
+            fileOutputProperties.partitionBy match {
+              case Some(_) =>
+                log.info("Recovering partitions")
+                catalog.recoverPartitions(tableName)
+              case _ => None
+            }
+            catalog.refreshTable(tableName)
```

Comment:
This will update the schema in hive?

Reply:
yeah it will, this way when writing a new table and it will update all partitions and refresh the metadata of the table in Hive